### PR TITLE
dyncom: Minor cleanup.

### PR DESCRIPTION
**Pull request by lioncash**

### CODE REVIEW

File: src/core/arm/dyncom/arm_dyncom_interpreter.cpp [hunk 1]
```diff
@@ -68,6 +69,67 @@ static void remove_exclusive(ARMul_State* state, ARMword addr){
...
+    int temp = 0;
+
+    switch (cond) {
+    case 0x0:
+        temp = ZFLAG;
+        break;
+    case 0x1: // NE
+        temp = !ZFLAG;
+        break;
+    case 0x6: // VS
```
Comment (linkmauve) [inline src/core/arm/dyncom/arm_dyncom_interpreter.cpp]: I know this was in the original code, but you should reorder them in ascending order.

Comment (lioncash) [inline src/core/arm/dyncom/arm_dyncom_interpreter.cpp]: I've amended the commit.

**Comment by yuriks** [comment]
LGTM

**Comment by bunnei** [comment]
LGTM

**Pull request merged**